js-i2b2/hive/i2b2_cell_communicator.js

```diff
@@ -193,9 +193,41 @@ i2b2.hive.communicatorFactory = function(cellCode){
 		console.groupEnd();
 		execBubble.timeSent = new Date();
 		commOptions.i2b2_execBubble = execBubble;
+		
```

Comment:
I don't understand what the rest of the patches to this file have to do with CAS.